File: Java/src/main/java/domain/Angajat.java
package domain;

public class Angajat extends Entity<String>{

    private String nume;
    private String prenume;
    private String parola;

    public Angajat(String nume, String prenume, String parola) {
        this.nume = nume;
        this.prenume = prenume;
        this.parola = parola;
    }

    public String getNume() {
        return nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public String getParola() {
        return parola;
    }
}

File: C#/Laborator2C/domain/Angajat.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lab2C.domain
{
    class Angajat:Entity<String>
    {
        public String Nume { get; set; }
        public String Prenume { get; set; }
        public String Parola { get; set; }

        public override string ToString()
        {
            return Nume+" "+Prenume;
        }
    }
}
